### ISSUE TITLE
Dead repo?

### ISSUE THREAD

**Issue by carlosbaraza** [yogiben/meteor-admin#274]
Hi managers,

This package is very interesting but I feel the repository moderators are not very active. There are multiple pending merge requests and minimal activity.

What could we do to reactivate the activity and the development?

Cheers,
Carlos.

**Comment by ertix4248** [comment]
it seems that there is a next version in planing/development 
https://github.com/meteor-factory/meteor-admin

unfortunately it is not very active as well

**Comment by ashishapy** [comment]
True, This is one of my favourite package and sad to know that it is not active.

Hope, there are any way to make it active. @yogiben

**Comment by carlosbaraza** [comment]
Maybe if @yogiben has no time to keep the repo active, someone could take over the responsibility and at least accept the merge requests and move the tickets.

**Comment by MartinBucko** [comment]
It's my favorite meteor repo also i will like to participate on it! @yogiben Please delegate what is posibble :)
I think this package can move over wordpress like admins in the future!

**Comment by macrozone** [comment]
I agree, we should take over the project. I am not sure if we should start with this repo here or the "next version" mentioned above.

**Comment by carlosbaraza** [comment]
That would be great actually. I am just concern about the fact that continuing on a different repo would confuse users.

@yogiben, could you give an opinion on the topic when you have some time?

Thank you.

**Comment by obiwarn** [comment]
That would be a great idea. We need a more community driven admin. And this one is the best to start with.

**Comment by mpowaga** [comment]
Hey @carlosbaraza, @paul-bo, @ashishapy, @macrozone, @obiwarn. Thanks for your interest in helping out. We just published the first version of [mfactory:admin](https://atmospherejs.com/mfactory/admin). It may be not feature rich and stable at this point but we hope to improve it soon. You can check out the roadmap on our trello [board](https://trello.com/b/QdYMgTRS/meteor-admin). Feel free to add your ideas.

Cheers.

**Comment by obiwarn** [comment]
This looks very promising. Thank you @mpowaga.

**Comment by macrozone** [comment]
@mpowaga Thank you very much!

Two things:

- is the api mfactory:admin more or less similar to yogiben:admin? Maybe add a bullet point to the readme for migration from yogiben:admin
- Add a notice to yogiben:admin at the beginning of the readme that it is deprecated and users should move to mfactory:admin. We need to focus the attention to the newer repo.

**Comment by jeffyamada** [comment]
@mpowaga I'm using yogiben:admin on a couple of production projects. Will anyone merge PRs in or is now a good time to fork?
